docs/specs/meta-clients/web3modal/api-v1-legacy.md

```diff
@@ -0,0 +1,142 @@
+# API
+
+Web3Modal has it's own api available at [api.web3modal.com](https://api.web3modal.com) that is used to fetch wallet data from a WalletConnect maintained [wallet & dapp registry](https://walletconnect.com/explorer).
+
+All responses are cached for a minimum of 2h.
+
+## Headers
+
+All api requests must contain following headers. See [types](./types.md)
+
+```ts
+interface Headers {
+  "x-project-id": string;
+  "x-sdk-type": SdkType;
+  "x-sdk-version": SdkVersion;
+}
+```
+
+## Methods
+
+### GET `/getWallets`
+
+Returns total count and wallet data, based on provided body inputs. See [types](./types.md).
+
+```ts
+interface Request {
+  headers: Headers;
+}
+
+interface UrlQueryParams {
+  page: string; // eg. 1
+  entries: string; // eg. 100
+  search?: string; // eg. MetaMa...
+  include?: string; // eg. id1,id2,id3
+  exclude?: string; // eg. id1,id2,id3
+  chains?: string; // eg. eip155:1,solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp
+  platform?: "ios" | "android";
+}
+
+interface Response {
```

Comment:
In practice how will this be used? When will it be called and with what parameters?

I.e. to get the first page of the modal it should only need to return no more than 5 entries, saving on bandwidth

Reply:
This is the legacy spec we are changing

Reply:
Nevermind